A/AM/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "AM"
```

Comment:
I take this package doesn't have a more descriptive name, does it?

Reply:
No, it does not have a more descriptive name.